gulpfile.js

```diff
@@ -497,7 +497,7 @@ gulp.task("ci:azure:get-extensions-version", (done) => {
     console.log(`Fetched ext version ${version} for ${extension}`);
     console.log(`##vso[task.setvariable variable=${extensionPrefix}_EXT_NAME]${extensionManifest}`);
     console.log(
-      `##vso[task.setvariable variable=${extensionPrefix}_VERSION;isOutput=true]${version}`,
+      `##vso[task.setvariable variable=${extensionPrefix}_VERSION;isOutput=true;isreadonly=true]${version}`,
```

Comment:
I was thinking it was getting overridden between stages, but as I found out, only the last stage is passed to the `stageDependencies`. I can omit this but in theory it is read only 🤷🏻 
https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-variables-scripts?view=azure-devops&tabs=bash#set-variable-properties